libs/vkd3d/state.c

```diff
@@ -1346,7 +1346,7 @@ struct vkd3d_render_pass_entry
     VkRenderPass vk_render_pass;
 };
 
-STATIC_ASSERT(sizeof(struct vkd3d_render_pass_key) == 48);
+STATIC_ASSERT(sizeof(struct vkd3d_render_pass_key) == 52);
```

Comment:
is there any particular reason we're insisting on a specific size here anyway?

Reply:
It's to ensure there is no packing padding, which would screw up with the key compare which is memcmp.

Reply:
added a comment